### ISSUE TITLE
Pretrained model evaluation

### ISSUE THREAD

**Issue by Matthewkm** [assembly-101/assembly101-action-recognition#10]
Hi and thanks for the dataset.

I'm running the evaluation code with the pre-trained model provided, however just utilising data from view v-2. 

On the validation data the model achieves an Overall Prec@1 = 38.84, which does not match the value of 40.6 provided in table 6 of your paper.

I just wanted to check if the values within table 6 calculated from the test set which we can not access? or if there is a problem with my decoded frames/data set pipeline?

Cheers, Matthew.

**Comment by assembly-101** [comment]
Hi,

The results provided in Table 6 of our main paper are indeed evaluated on the test set. We have observed a trend in the accuracy on the full test set (for TSM) being generally around 1% higher than the validation set.

We withhold the ground truth action labels for the test split. However, you can generate (verb/object/action) accuracy for the test set  by submitting your prediction `.json` file to our [codalab platform](https://codalab.lisn.upsaclay.fr/competitions/5256).

Best,
Assembly101 Team
